NimbleMedium/Sources/Presentation/Modules/EditArticle/EditArticleView.swift

```diff
@@ -13,12 +13,16 @@ struct EditArticleView: View {
 
     @Environment(\.presentationMode) var presentationMode
 
+    @ObservedViewModel private var viewModel: EditArticleViewModelProtocol
```

Comment:
I think we can just do this:
```suggestion
    @ObservedViewModel private var viewModel: EditArticleViewModelProtocol = Resolver.resolve()
```
and remove the `init()` function completely, what do you think? 🙏